docs/pages/careers.tsx

```diff
@@ -28,18 +28,6 @@ const openRolesData = [
           'You will provide guidance to the community and solve their struggle, working on one of the most popular React UI library.',
         url: '/careers/react-engineer-material-ui-maintainer/',
       },
-      {
-        title: 'React Engineer — Docs-infra',
```

Comment:
We tend to comment those out for future convince but I'm ok removing them as well. I leave it up to you.
The same comment applies to the other role entries.

Reply:
That makes sense for roles we frequently open, I don't expect that for these role though :)